resources/js/Pages/Vault/Contact/Index.vue

```diff
@@ -39,6 +68,31 @@
                 {{ $t('No labels yet.') }}
               </p>
             </div>
+
+            <!-- sort -->
+            <div>
+              <div class="mb-3 border-b border-gray-200 dark:border-gray-700">
+                <span class="mr-1"> 🌡️ </span>
+                {{ $t('Sort contacts') }}
+              </div>
+              <ul>
+                <li class="mb-1">
+                  <span @click="update('last_updated')" class="cursor-pointer text-blue-500 hover:underline">{{
+                    $t('By last updated')
+                  }}</span>
+                </li>
+                <li class="mb-1">
+                  <span @click="update('a_to_z')" class="cursor-pointer text-blue-500 hover:underline">{{
+                    $t('From A to Z')
+                  }}</span>
+                </li>
+                <li class="mb-1">
+                  <span @click="update('z_to_a')" class="cursor-pointer text-blue-500 hover:underline">{{
+                    $t('From Z to A')
+                  }}</span>
+                </li>
+              </ul>
```

Comment:
The UX is a little odd. I think it would be clearer if it was a dropdown list or something else. Here the user can't tell which is the current selected sorting.

Reply:
changed